opm/parser/eclipse/EclipseState/Tables/Tables.cpp

```diff
@@ -1085,6 +1085,22 @@ struct flat_props< PVTWRecord, 0 > {
     }
 };
 
+constexpr const char* pvcdo_err[] = {
```

Comment:
I agree somewhat with Andreas here, but I want the most readable solution, and I don't immediately see the consequences of Andreas' solution.

Chose the one that you think will be the most readable (read: maintainable).